ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryConsistencyFunctionalTest.java

```diff
@@ -469,4 +481,31 @@ public void configure(Map<String, ?> map) {
       return CompletableFuture.completedFuture(null);
     }
   }
+
+  public static class StaticStreamsTaskAssignor implements TaskAssignor {
```

Comment:
nit: can we reuse the class already created in https://github.com/confluentinc/ksql/pull/8351/?

Reply:
+1 :)